.python-version

```diff
@@ -0,0 +1 @@
+renuo-thumbor-2.7.9
```

Comment:
Is this right? I cannot see the possibility to put it like that according to the documentation of [pyenv](https://github.com/yyuu/pyenv).